sdk/nodejs/runtime/closure/codePaths.ts

```diff
@@ -184,7 +185,7 @@ function allFoldersForPackages(
 
                     // From: https://github.com/npm/read-package-tree/blob/5245c6e50d7f46ae65191782622ec75bbe80561d/rpt.js#L121
                     root.package = computeDependenciesDirectlyFromPackageFile(
-                        filepath.join(root.realpath, "package.json"), logResource);
+                        upath.join(root.realpath, "package.json"), logResource);
```

Comment:
What's the difference between `upath.join` used here and `upath.joinSafe` used on line 319?

Reply:
I meant to use join, not joinSafe.  joinsSafe is interesting, in that it preserves a leading "./" if there is one.  However, we didn't need that behavior before with path.join (which does not do that), so i'm going to preserve the path.join behavior since it wasn't a problem before.

Reply:
Actually, have tweaked things.  worth looking again.

Reply:
gotcha - thanks for the explanation.